flight/Modules/Autotune/autotune.c

```diff
@@ -207,25 +273,21 @@ static void AutotuneTask(void *parameters)
 			continue;
 		}
 
-		float throttle;
-
-		ManualControlCommandThrottleGet(&throttle);
-				
 		switch(state) {
 			case AT_INIT:
-
 				last_update_time = PIOS_Thread_Systime();
 
 				// Only start when armed and flying
-				if (flightStatus.Armed == FLIGHTSTATUS_ARMED_ARMED && throttle > 0) {
+				if (flightStatus.Armed == FLIGHTSTATUS_ARMED_ARMED) {
```

Comment:
You removed the throttle>0 from here which looks like prevented the system from gathering points while armed and not flying, was that on purpose? If so, are there any downsides? Were there little stray points in general?